apps/barista/src/styles/global/_base.scss

```
@@ -82,3 +82,81 @@ mark {
 code {
   font-family: 'Bitstream Vera', monospace;
 }
+
+.ba-single-page-content {
```

```suggestion
.ba-single-page-content > {
```
We should have a direct child selector here otherwise we bleed into the examples